### PR TITLE
Docs: Add PBFT steps to K8s network procedure

### PR DESCRIPTION
**Pull request by chenette**
This PR updates "Using Kubernetes for a Sawtooth Test Network" in the App Dev Guide
to add PBFT steps and related information (intro material, overviews, links, etc.) 
alongside the existing PoET steps.
 
It also contains changes to make the Docker, Ubuntu, and Kubernetes procedures consistent.

### CODE REVIEW

File: docs/source/app_developers_guide/kubernetes_test_network.rst [hunk 1]
```diff
@@ -132,67 +119,172 @@ information, see the
         && chmod +x minikube && sudo mv minikube /usr/local/bin/
 
 
-Step 2: Start and Test Minikube
--------------------------------
+Step 2: Start Minicube
```
Comment (rbuysse) [inline docs/source/app_developers_guide/kubernetes_test_network.rst]: Minikube

File: docs/source/app_developers_guide/kubernetes_test_network.rst [hunk 2]
```diff
@@ -132,67 +119,172 @@ information, see the
...
+----------------------
 
-This step summarizes the procedure to start Minikube and test basic
-functionality. If you have problems, see the Kubernetes document
-`Running Kubernetes Locally via Minikube
-<https://kubernetes.io/docs/setup/minikube/>`_.
-
-1. Start Minikube.
+1. Start Minicube.
```
Comment (rbuysse) [inline docs/source/app_developers_guide/kubernetes_test_network.rst]: Minikube

File: docs/source/app_developers_guide/kubernetes_test_network.rst (hunk 1 above)
Comment (chenette) [reply docs/source/app_developers_guide/kubernetes_test_network.rst]: Fixed.

File: docs/source/app_developers_guide/kubernetes_test_network.rst (hunk 2 above)
Comment (chenette) [reply docs/source/app_developers_guide/kubernetes_test_network.rst]: Fixed.

File: docs/source/app_developers_guide/kubernetes_test_network.rst [hunk 3]
```diff
@@ -132,67 +119,172 @@ information, see the
...
+Step 4: (PBFT Only) Configure Keys for the Kubernetes Pods
+----------------------------------------------------------
+
+.. important::
+
+   Skip this step if you are using PoET consensus.
+
+For a network using PBFT consensus, this step generates public and private
+validator keys for each pod in the network, then creates a Kubernetes ConfigMap
+so that the Sawtooth network can use these keys.
```
Comment (ltseeley) [inline docs/source/app_developers_guide/kubernetes_test_network.rst]: Not a necessity (and maybe TMI for this guide), but you could maybe explain why this step is necessary for PBFT (members list needs to be known before genesis).

Comment (chenette) [reply docs/source/app_developers_guide/kubernetes_test_network.rst]: I like it.  I just pushed a fixup commit with this change (and a bunch of typo fixes and wording tweaks elsewhere in the section).

**Pull request merged**